keep/contextmanager/contextmanager.py

```diff
@@ -106,6 +107,9 @@ def get_full_context(
             "alert": self.event_context,  # this is an alias so workflows will be able to use alert.source
         }
 
+        if isinstance(full_context['alert'], dict):
+            full_context['alert'] = AlertDto(**full_context['alert'])
```

Comment:
Wonder if that's a correct place to cast. Casting is required because sometimes it's an AlertDto, sometimes a dict (mainual/automatic triggering) which complicate provider code.

Reply:
@shahargl fixed this bug a few hours ago~ (it should always be `AlertDto` now)